mantidproject/mantid
# Snap reduce

This is bringing `/SNS/SNAP/.../SNAP_Reduce.py` into mantid. It should generate the same results. One notable difference is the instrument is replaced in the final workspace with a simplified one.

**To test:**

Try reducing various SNAP files using different options. Make sure to run the system test with `--archivesearch` otherwise it will be skipped.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
